composer.json

```diff
@@ -13,12 +13,27 @@
         "php": "^7.1 || 8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7 || ^8 || ^9",
-        "squizlabs/php_codesniffer": "^3.5"
+        "phpunit\/phpunit": "^7 || ^8 || ^9",
+        "squizlabs\/php_codesniffer": "^3.5",
+        "suilven\/php-travis-enhancer": "^1.0",
```

Comment:
Once travis has been enhanced, can this package be removed? Or does it do something beyond introducing the changes in this PR?

Reply:
I forgot to remove it

Reply:
Sweet as, I removed it in my final PR>